### DISCUSSION TITLE
Production ready ?

### DISCUSSION THREAD

**Discussion by aqueos** [mydumper/mydumper#1696] [General]
hello there,

  I was using mydumper back in 0.12 days, then it started skipping tables and having a lots of issue so i stopped used it and reverted to mysqldump.

  Now, 2025 is near and i wanted to ask the community what they think about the state of the tool:

1/ is it reliable enough now, do you trust it with essential data ?
2/ is the export format stable so we can import/export with different version without being left with unusable dumps ?
3/ will it release in 1.x  ? That should say that the devs are trusting the 1/ and 2/ 

 Hope  you can enlighten me on the state of the project :)

best regards,
Ghislain.

**Comment by davidducos** [comment]
Hi @aqueos 

I might not be the person to give an objective point of view, but anyways

1- there are plenty of successful cases that are not visible and there are a lot of issues that has been resolved since v0.12. I would say that it depends on the use case, the features that you are using and if you are reaching some edge case. 
Take into account also that we have to made some changes as earlier versions of mydumper were not able to handle some of the most important use cases.

2- the export format has been estable in the latest versions and I'm confident that it will not change as nowadays the metadata file can be read from myloader.

Regards
